tap_hibob/streams.py

```diff
@@ -10,15 +10,22 @@
 # TODO: Delete this is if not using json files for schema definition
 SCHEMAS_DIR = Path(__file__).parent / Path("./schemas")
 
-
+from tap_hibob.schemas import Employees
 
 class EmployeesStream(HibobStream):
+    _LOG_REQUEST_METRIC_URLS=True
```

Comment:
What does this do?

Reply:
It's a logging practice; thanks for spotting it, I need to remove it